03_analysis/compare_ffs.py

```diff
@@ -396,53 +457,62 @@ def label(x, color, label):
         ax = plt.gca()
 
         # set axis font size
-        if what_for == 'paper': fs = 8
+        if what_for == 'paper': fs = 14
         elif what_for == 'talk': fs = 14
 
         ax.text(0, .2, label, fontweight="bold", color=color, fontsize=fs,
                 ha="left", va="center", transform=ax.transAxes)
 
     if what_for == 'paper':
         ridgedict = {
-            "h":0.45,
-            "lw":1.0,
-            "vl":0.1,
-            "xfontsize":10,
+            "h":0.9,
+            "lw":2.0,
+            "vl":1.0,
+            "xfontsize":14,
         }
     elif what_for == 'talk':
         ridgedict = {
-            "h":1.0,
-            "lw":1.5,
-            "vl":0.5,
+            "h":2.0,
+            "lw":3.0,
+            "vl":1.0,
             "xfontsize":16,
         }
 
     num_methods = len(mydata)
 
+    # Initialize the FacetGrid object
+    my_cmap = "tab10"
+    pal = sns.palplot(sns.color_palette(my_cmap))
+    colors = sns.color_palette(my_cmap)
+    all_labels = ['MMFF94S', 'GAFF2', 'OPLS3e', 'OpenFF-1.2', 'OpenFF-1.0', 'Smirnoff99Frosst', 'MMFF94', 'GAFF',  'B3LYP-D3BJ/DZVP', 'OpenFF-1.1']
```

Comment:
Same comment as above regarding hard-coded labels. Shouldn't the labels be read in from the input file automatically?

Reply:
See comment below. Resolved in b59b5b5